Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Microsoft.VisualBasic.FileIO;
+using System.Collections.Generic;
+
+namespace SupportBank
+{
+    class Program
+    {
+        static Dictionary<string, Account> ReadFile() {
+
+            var accounts = new Dictionary<string, Account>();
+
+            var parser = new TextFieldParser(@"Transactions2014.csv");
+            parser.TextFieldType = FieldType.Delimited;
+            parser.SetDelimiters(",");
+
+            if (!parser.EndOfData)
+                parser.ReadLine();
+
+            while (!parser.EndOfData)
+            {
+                var fields = parser.ReadFields();
+                var newTrans = new Transaction(fields);
+
+                foreach (string name in fields[1..3])
```

Comment:
So is this just looping over two things? In that case I would just repeat it. I think using a `foreach` makes the code a bit harder to read, as it's a bit unexpected. Therefore it's not worth the lines of code you'd save (which shouldn't be that much anyway)? 🙂